pkgs/top-level/all-packages.nix

```diff
@@ -13388,6 +13388,8 @@ let
 
   viber = callPackage ../applications/networking/instant-messengers/viber { };
 
+  wpsoffice = callPackage ../applications/office/wpsoffice {};
```

Comment:
we try to keep packages in an alphabetical order. can you put it near the other w* ?